crates/ruff/src/visibility.rs

```diff
@@ -77,13 +78,26 @@ pub fn is_abstract(checker: &Checker, decorator_list: &[Expr]) -> bool {
 }
 
 /// Returns `true` if a function definition is a `@property`.
-pub fn is_property(checker: &Checker, decorator_list: &[Expr]) -> bool {
+/// `extra_properties` can be used to check additional non-standard
+/// `@property`-like decorators.
+pub fn is_property(
+    checker: &Checker,
+    decorator_list: &[Expr],
+    extra_properties: Option<&[CallPath]>,
```

Comment:
It's preferable to just take `&[CallPath]` here, since `extra_properties.iter().any(...)` should return `false` if it's empty anyway.

Reply:
Sure!

Reply:
Thanks :)